### DISCUSSION TITLE
I dont have -o --output in my teler

### DISCUSSION THREAD

**Discussion by JRockiller1** [teler-sh/teler#249] [Q&A]
Want to use fail2ban and teler but -o --output is not defined or recognised when i run any command with it.

tail -f /var/log/nginx/access.log | teler -c teler.example.yaml -o /var/log/teler/teler-threats.log
teler -i /var/log/nginx/access.log -o /var/log/teler/teler-threats.log

None of this works however if i remove the -o, everything works as per normal. Just wondering why -o, --output does not exist for me

**Comment by dwisiswant0** [comment]
Which version did you use? As per [documentation](https://teler.app/configuration#logs), output is defined under configuration file.
